lesson_22/script.js

```diff
@@ -0,0 +1,23 @@
+console.log('Hello world!!!');
+let message = 'Yulian';
+alert('Hi, ' + message + '!!!');
+
+let a = +prompt("Enter number five");
+let b = +prompt("Enter number six");
+if (b===0) {
+    alert ("Error:division by zero");
+} else {
+    alert(`Sum = `+ ( a + b ));
```

Comment:
if you use ` you don't have to use + in building you string:

it's enough: `Sum ${a+ b}`